README.md

```diff
@@ -144,6 +144,12 @@ $ cd serialportstream/dll/serialunix
 $ ./build.sh
 ```
 
+for __OSX__ you must rename your library
```

Comment:
Do you know what needs to be done in CMake that this step isn't necessary? What does a "make install" do?